commit/action.yml

```diff
@@ -88,18 +93,34 @@ runs:
         token: ${{ inputs.token }}
         persist-credentials: true
 
+    # install asdf itself
+    # https://github.com/asdf-vm/actions/blob/master/setup/action.yml
```

Comment:
Please remove these comments, they don't add any value in my opinion (moreover the links aren't fixed because they point to `master`).